templates/status/500.tmpl

```diff
@@ -3,9 +3,9 @@
 	<p style="margin-top: 100px"><img class="ui centered image" src="{{StaticUrlPrefix}}/img/500.png" alt="500"/></p>
 	<div class="ui divider"></div>
 	<br>
-	{{if .ErrorMsg}}<p>An error has occurred :</p>
+	{{if .ErrorMsg}}<p>{{.i18n.Tr "error.occurred"}}:</p>
 	<pre style="text-align: left">{{.ErrorMsg}}</pre>{{end}}
-	{{if .ShowFooterVersion}}<p>Application Version: {{AppVer}}</p>{{end}}
-	{{if .IsAdmin}}<p>If you are sure this is Gitea bug, please search for issue on <a href="https://github.com/go-gitea/gitea/issues">GitHub</a> and open new issue if necessary.</p>{{end}}
+	{{if .ShowFooterVersion}}<p>{{.i18n.Tr "admin.config.app_ver"}}: {{AppVer}}</p>{{end}}
+	{{if .IsAdmin}}<p>{{.i18n.Tr "error.report_message"}}</p>{{end}}
```

Comment:
```suggestion
	{{if .IsAdmin}}<p>{{.i18n.Tr "error.report_message" | Safe}}</p>{{end}}
```

Reply:
thought about that already - but is the link to the bug tracker klickable afterwards?

Reply:
I think it's the opposite: `Safe` will mark the text as safe and _skip_ sanitization. 🤔 
https://golang.org/pkg/html/template/#HTML